### PR TITLE
use flexbox, and a lot of other shit

### PR DESCRIPTION
**Pull request by akisblack**

### CODE REVIEW

File: src/lib/InfoCard.svelte [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    export let uploadedOn;
+</script>
+
+<svelte:head>
+    <title>{title} | Odyssey's blog</title>
+</svelte:head>
+
+<div class="container">
+    <div class="card">
+        <a href="/">Back to main page where all the posts are.</a>
```
Comment (Odyssey346) [inline src/lib/InfoCard.svelte:14]: Keep this in mdsvex_content/where it originally was. Let me know if you can't

File: src/routes/posts/helloworld.md [hunk 2]
```diff
@@ -12,4 +12,4 @@ uploadedon: 2.03.2021
 
 I finally did it. I made my blog. I'm happy now.
 
-I still have a lot to work on. In the mean time, check out the GitHub <a href="https://github.com/Odyssey346/blog">repository.</a>
\ No newline at end of file
+I still have a lot to work on. In the mean time, check out the GitHub <a href="https://github.com/Odyssey346/blog">repository</a>.
```
Comment (Odyssey346) [inline src/routes/posts/helloworld.md:15]: No

File: svelte.config.js [hunk 3]
```diff
@@ -7,20 +6,20 @@ const dev = process.env.NODE_ENV === 'development';
...
-		layout: "./src/lib/DescCard.svelte"
+		layout: "./src/lib/InfoCard.svelte"
 	})],
 
 	kit: {
 		adapter: adapter(),
 		paths: {
-			base: '/blog',
-		},
+			base: dev ? '' : '/blog',
```
Comment (Odyssey346) [inline svelte.config.js:21]: Does it work now btw?

File: svelte.config.js [hunk 4]
```diff
@@ -1,4 +1,3 @@
-import mdsvexConfig from './mdsvex.config.js';
```
Comment (Odyssey346) [inline svelte.config.js:1]: Oh btw why'd you remove this file?

**Review by Odyssey346 — CHANGES_REQUESTED** [review]
some minor stuff

File: svelte.config.js (hunk 3 above)
Comment (akisblack) [reply svelte.config.js:21]: it should

File: svelte.config.js (hunk 4 above)
Comment (akisblack) [reply svelte.config.js:1]: it isnt needed, all the configs are now loaded from svelte config

Comment (Odyssey346) [reply svelte.config.js:1]: Alright gotcha, I see.

**Comment by Odyssey346** [comment]
LGTM + Reviewed + No Issues + Code + Ship

**Pull request merged**